app/travel_analytics/webapp/localService/metadata.xml

```diff
@@ -0,0 +1,130 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Why isn't that generated by us as part of `2edmx`?

Reply:
file seems to be unnecessary, not presetn in travel_processor -> removed